db/migrate/20250827134624_delete_history_page_documents_and_editions.rb

```diff
@@ -0,0 +1,19 @@
+# This will delete all editions with the following base paths, as well as their parent documents:
+#       /government/history/10-downing-street
+#       /government/history/11-downing-street
+#       /government/history/king-charles-street
+#       /government/history/lancaster-house
+#       /government/history/1-horse-guards-road
+#       /government/history
```

Comment:
I see this is now in scope for https://gov-uk.atlassian.net/browse/WHIT-2374 - nice!